src/deb/control/control

```diff
@@ -1,6 +1,6 @@
 Package: [[artifactId]]
 Version: [[version]]
-Description: [[name]]
+Description: [[name]]+[[build.timestamp]]-plos[[build.counter]]
 Architecture: all
```

Comment:
Oops – my bad. Thanks for catching that; will change it to use the version field shortly (for both NED and NED-CAS).